en/field-work/map-interaction.rst

```diff
@@ -0,0 +1,34 @@
+Interact with the map
+#####################
+
+Map legend
+==========
+Open the **Main menu** and expend the layers list to display the legend of the map.
+
+.. image:: ../images/user-guide_legend.png
+
+
+
+Identify features
```

Comment:
We now have the possibility to "select" and "multi edit". This starts from "identify features". Do we want to mention this in here or probably have a new section below?
Open an issue for that?

Reply:
open an issue.